app/views/general/_split_button_checkbox.html.erb

```diff
@@ -0,0 +1,53 @@
+<%
+  #<%= render :partial => 'general/split_button_checkbox',
+  #locals: { checkbox_id: publishing_item_param(item),
+  #          checkbox_class: "#{item.class.name}_#{item.id}",
+  #          published: true,
+  #          checkbox_text: "Publish?",
+  #          checked: checked,
+  #          toggle: children.any?,
+  #          extra_nesting: 1 } - % >
+
+  checkbox_class ||= "no_name"   # <- Needed!
+  checkbox_id ||= checkbox_class
+  published ||= false
+  checkbox_text ||= ""
+  checked ||= false
+  checked = !([false,'false',0,'0','no'].include? checked)
```

Comment:
Do we need this? Can we just make sure all usage of this partial pass a boolean value?